### PR TITLE
@kanaabe => Fix CanvasDisplayVideo click-out

### PR DESCRIPTION
**Pull request by damassi**
Address https://github.com/artsy/publishing/issues/206

### CODE REVIEW

File: src/Components/Publishing/Display/Canvas/CanvasContainer.tsx [hunk 1]
```diff
@@ -50,7 +51,14 @@ export class CanvasContainerComponent extends React.Component<CanvasContainerPro
   }))
   openLink(e) {
     e.preventDefault()
-    if (!e.target.className.includes('CanvasVideo')) {
+    const videoClasses = [
+      'VideoControls__PlayButton',
+      'VideoControls__PlayButtonCaret',
+      'CanvasVideo__video'
+    ]
+    const isVideoClickArea = videoClasses.some(c => e.target.className.includes(c))
```
Comment (damassi) [inline src/Components/Publishing/Display/Canvas/CanvasContainer.tsx]: Within the current context, I'm not sure of a better way to do this.

Comment (kanaabe) [reply src/Components/Publishing/Display/Canvas/CanvasContainer.tsx]: Hmm could z-index or absolute positioning work here?

Comment (damassi) [reply src/Components/Publishing/Display/Canvas/CanvasContainer.tsx]: The problem is that there is an outer `a` tag that then contains `n` kinds of elements (cover, bottom text area, video, caret, etc), in a dynamic way -- without blacklisting specific elements with classes that can then be filtered against there's no way to accurately know whats being clicked.

**Comment by kanaabe** [comment]
Alrighty, LGTM!

**Pull request merged**